kvmagent/kvmagent/plugins/tensorfusion/monitor.py

```
@@ -200,10 +207,10 @@ def _do_restart(self, w, backoff):
             # Reap the old dead process to prevent zombie.
             if getattr(w, 'pid', None):
                 try:
-                    self._executor.reap_dead(w.pid)
+                    self._executor.reap_dead(w)
                 except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)